packages/toolboxes/cosmos/src/thorchainUtils/memo.ts

```diff
@@ -0,0 +1,26 @@
+import { MemoAction } from "./types";
+import type { MemoInfo, MemoSwapInfo } from "./types/memo";
+
+export function parseMemo(memo: string): MemoInfo {
```

Comment:
whats the idea here @olegpetroveth ? currently this is just throwing errors.

also, what do we need shortestIdentifier for? I think we should not add those into asset value but rather have a helper that matches it into identifier, wdyt?